### PR TITLE
add payload to delete and get

### PR DESCRIPTION
**Pull request by markGilchrist**

### CODE REVIEW

**Comment by markGilchrist** [comment]
Line 456, of RequestTest.kt adds both query params and body, please let me know if this is not a suitable test

File: fuel/src/test/kotlin/com/github/kittinunf/fuel/RequestTest.kt [hunk 1]
```diff
@@ -735,10 +741,10 @@ class RequestTest : BaseTestCase() {
         request.responseString().third.fold({ string ->
             try {
                 val json = JSONObject(string)
-                assertEquals(json.getJSONObject("args").getString("bar"),lionel)
+                assertEquals(json.getJSONObject("args").getString("bar"), lionel)
                 assertEquals(json.getJSONObject("args").getJSONArray("foo[]").map { it.toString() },
-                list.toList())
-            } catch (e: Exception){
+                        list.toList())
```
Comment (kittinunf) [inline fuel/src/test/kotlin/com/github/kittinunf/fuel/RequestTest.kt]: [Super nits] Can we combine this with the previous line?

**Comment by kittinunf** [comment]
Ah, I got it. Thanks. I left one small note, otherwise it is good.

File: fuel/src/test/kotlin/com/github/kittinunf/fuel/RequestTest.kt (hunk 1 above)
Comment (markGilchrist) [reply fuel/src/test/kotlin/com/github/kittinunf/fuel/RequestTest.kt]: done

**Pull request merged**